calico.yaml

```diff
@@ -0,0 +1,371 @@
+                   "${{targets.subpkgdir}}"/licenses \
+                   "${{targets.subpkgdir}}"/usr/sbin
+
+          cp -a node/filesystem/etc/ "${{targets.subpkgdir}}"/etc
+          cp -a node/filesystem/included-source/ "${{targets.subpkgdir}}"/included-source
+          cp -a node/filesystem/licenses/ "${{targets.subpkgdir}}"/licenses
+          cp -a node/filesystem/usr/ "${{targets.subpkgdir}}"/usr
+          cp -a node/filesystem/sbin/* "${{targets.subpkgdir}}"/usr/sbin/
+
+          cp ./node/LICENSE "${{targets.subpkgdir}}"/licenses
+
+  # The felix binaries aren't actually shipped, but they contain some useful testing utilities
+  - name: "calico-felix"
+    dependencies:
+      runtime:
+        - iproute2
+        - libelf
+        - zlib
+        - libbpf
+    pipeline:
+      - working-directory: felix
+      - runs: |
+          _arch=$(uname -m); case $_arch in aarch64) _arch="arm64" ;; x86_64) _arch="amd64" ;; esac
+
+          LIBBPF_CONTAINER_PATH=$(pwd)/felix/bpf-gpl/include/libbpf/src
+          BPFGPL_CONTAINER_PATH=$(pwd)/felix/bpf-gpl
+
+          CGO_LDFLAGS="-L$LIBBPF_CONTAINER_PATH/$_arch -lbpf -lelf -lz"
+          CGO_CFLAGS="-I$LIBBPF_CONTAINER_PATH -I$BPFGPL_CONTAINER_PATH"
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/felix \
+            ./felix/cmd/calico-felix/calico-felix.go
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/calico-bpf \
+            ./felix/cmd/calico-bpf/main.go
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/pktgen \
+            ./felix/fv/pktgen/pktgen.go
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/iptables-locker \
+            ./felix/fv/iptables-locker/iptables-locker.go
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/test-connection \
+            ./felix/fv/test-connection/test-connection.go
+
+          CGO_ENABLED=1 \
+            CGO_LDFLAGS="$CGO_LDFLAGS" \
+            CGO_CFLAGS="$CGO_CFLAGS" \
+            go build -v -tags cgo -v -buildvcs=false \
+            -o "${{targets.subpkgdir}}"/usr/bin/test-workload \
+            ./felix/fv/test-workload/test-workload.go
+
+  - name: "calico-cni"
+    pipeline:
+      - uses: go/build
```

Comment:
Can you do a pass and make sure all of the `go/build` steps have `ldflags: -s -w`? 🙏 

see also: https://github.com/wolfi-dev/os/pull/2377